database/migrations/audits.stub

```diff
@@ -28,6 +28,7 @@ class CreateAuditsTable extends Migration
         Schema::create('audits', function (Blueprint $table) {
             $table->increments('id');
             $table->unsignedInteger('user_id')->nullable();
+            //$table->string('user_type')->nullable();
```

Comment:
This should not be commented, since we want the column.

Reply:
The reason that I left it commented out is because you don't need it for the 'normal' mode, just when you need it morphable. Still want me to uncomment this?

Reply:
Yup